CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_github_source v0.5.0
+
+## 🚨  Breaking Changes 🚨 
+- Addition of the `label` source table. This was necessary as the `issue_label` depreciated the `label` column. The respective label information will now be synced within the `label` table. ([#22](https://github.com/fivetran/dbt_github_source/pull/22))
```

Comment:
also might wanna add the same note about kicking off historical resyncs in the source changelog?

Reply:
Great idea!